moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -52,11 +52,13 @@ constexpr double DEFAULT_MAX_ACCELERATION = 10;  // rad/s^2
 constexpr double DEFAULT_MAX_JERK = 1000;        // rad/s^3
 constexpr double MAX_DURATION_EXTENSION_FACTOR = 10.0;
 constexpr double DURATION_EXTENSION_FRACTION = 1.1;
+constexpr double OVERSHOOT_CHECK_PERIOD = 0.01;  // sec
```

Comment:
Can you add a comment on what this magic number does and why it is necessary? Is it useful to make it a parameterizable?

Reply:
done

Reply:
i'm reluctant to add a parameter because --

- there are none so far
- i think this option will be used rarely
- we could add it later if need arises